auto_nag/scripts/fuzz_blockers.py

```diff
@@ -0,0 +1,78 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.history import History
+from auto_nag.nag_me import Nag
+
+# TODO: should be moved when resolving https://github.com/mozilla/relman-auto-nag/issues/1384
+LOW_SEVERITY = ["S3", "normal", "S4", "minor", "trivial", "enhancement"]
+
+
+class FuzzBlockers(BzCleaner, Nag):
+    def __init__(self, waiting_days: int = 3):
+        """Constructor
+
+        Args:
+            waiting_days: number of days to wait after the bug creation before
+                starting to nag.
+        """
+        super().__init__()
+
+        self.waiting_days = waiting_days
+
+    def description(self):
+        return "Bugs that prevent fuzzing from making progress"
+
+    def nag_template(self):
+        return super().template()
+
+    def set_people_to_nag(self, bug, buginfo):
+        persons = [
+            bug["assigned_to"],
+            bug["triage_owner"],
+        ]
+        if not self.add(persons, buginfo):
+            self.add_no_manager(buginfo["id"])
+
+        return bug
+
+    def get_mail_to_auto_ni(self, bug):
+        if bug["severity"] in LOW_SEVERITY and not self._is_commented(bug):
+            return utils.get_mail_to_ni(bug)
+
+    @staticmethod
+    def _is_commented(bug: dict) -> bool:
+        """Get whether the bug has a previous comment by this tool"""
+        for comment in reversed(bug["comments"]):
+            if comment["creator"] == History.BOT and comment["raw_text"].startswith(
+                "This bug prevent fuzzing from making progress."
```

Comment:
```suggestion
                "This bug prevents fuzzing from making progress."
```

Lately I'm thinking that these checks are very brittle (e.g. if we change the message in any way, we have to remember to update the search too)

Reply:
Even updating it could not be enough sometimes (if we want to support old comments). I will file an issue to follow up on this.

Reply:
Filed: https://github.com/mozilla/relman-auto-nag/issues/1577